experimental/PropertyDDS/packages/property-inspector-table/src/InspectorTableTypes.ts

```diff
@@ -5,8 +5,77 @@
 
 import { BaseProxifiedProperty } from "@fluid-experimental/property-proxy";
 import { BaseProperty } from "@fluid-experimental/property-properties";
+
+import { BaseTableProps, SortOrder } from "react-base-table";
 import { IRepoExpiryGetter, IRepoExpirySetter } from "./CommonTypes";
-import { IInspectorSearchState } from "./utils";
+
+export interface SearchResult {
+  foundMatches?: IInspectorSearchMatch[];
+  matchesMap?: IInspectorSearchMatchMap;
+  currentResult?: number;
+}
+export interface IShowNextResultResult {
+  /**
+   * New rows needed to expand to show next results
+   */
+  expandedRows: IExpandedMap;
+  /**
+   * row index of the next result. This value is used to automatically scroll in the table
+   */
+  rowIdx: number;
+  /**
+   * Next result column index
+   */
+  columnIdx: number;
+}
+
+interface ISearchLevelState {
+  data?: IInspectorRow[];
+  index: number;
+}
+
+export interface IInspectorSearchState {
```

Comment:
Maybe add `REVISIT` for search, in general? As it is still tightly coupled to a `propertyDDS` implementation.